heimdall-middleware-spec/src/test/java/br/com/conductor/heimdall/middleware/util/helpermock/call/RequestMockTest.java

```diff
@@ -0,0 +1,86 @@
+package br.com.conductor.heimdall.middleware.util.helpermock.call;
+
+import br.com.conductor.heimdall.middleware.spec.Header;
+import br.com.conductor.heimdall.middleware.spec.Query;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.Timeout;
+
+import java.util.HashMap;
+
+public class RequestMockTest {
+
+  @Rule public final ExpectedException thrown = ExpectedException.none();
+  @Rule public final Timeout globalTimeout = new Timeout(10000);
+
+  @Test
+  public void headerOutputNotNull() {
+
+    final RequestMock objectUnderTest = new RequestMock();
+
+    final Header retval = objectUnderTest.header();
+
+    Assert.assertNotNull(retval);
+    final HashMap<String, String> hashMap = new HashMap<String, String>();
+    Assert.assertEquals(hashMap, retval.getAll());
+  }
+
+  @Test
+  public void queryOutputNotNull() {
+
+    final RequestMock objectUnderTest = new RequestMock();
+
+    final Query retval = objectUnderTest.query();
+
+    Assert.assertNotNull(retval);
+    final HashMap<String, String> hashMap = new HashMap<String, String>();
```

Comment:
Since its an empty hashmap you could do
`Assert.assertEquals(new HashMap<String, String>(), retval.getAll());`

and get rid of this line` final HashMap<String, String> hashMap = new HashMap<String, String>();`